src/utils/timeSeriesResultsByOption.js

```diff
@@ -1,147 +1,116 @@
-import * as _ from 'lodash'
-import { FV, PMT } from 'formulajs/lib/financial'
-
 const COMPOUND_FREQUENCY = 12
 
-/**
- * @param {number} option.loanAmt How much money the loan is
- * @param {number} option.investmentRate Market return rate (in decimals)
- * @param {number} option.inflationRate Annual inflation rate (in decimals)
- * @param {object} option.shorterOption
- *   @property {number} mortgageRate Mortgage APR (in decimals)
- *   @property {number} mortgageTerm Mortgage term (in years)
- * @param {object} option.longerOption
- *
- * @return {object}
- *   @property {object[]} shorter
- *     @property {number} pmt Monthly payment amount (for that year)
- *     @property {number} loanAmt Remaining loan amount
- *     @property {number} investmentAmount How much money in investments
- *   @property {object[]} longer
- */
-export function getMonthly ({
```

Comment:
Why remove the type definition?

Reply:
i just deleted everything to start from scratch, will add back